1-exercises/E-for-loop/exercise2.js

```diff
@@ -26,8 +26,10 @@ const AGES = [
     49
 ];
 
-// TODO - Write for loop code here
+for(let i=0;i<WRITERS.length;i++){
 
+    console.log(`${WRITERS[i]} is ${AGES[i]} old`) // TODO - Write for loop code here
```

Comment:
Nice work using Backticks helps to include variables in a string (interpolation)